doc/classes/Image.xml

```diff
@@ -18,6 +18,7 @@
 			<param index="1" name="contrast" type="float" />
 			<param index="2" name="saturation" type="float" />
 			<description>
+				Adjusts the image's [param brightness], [param contrast] and [param saturation].
```

Comment:
Would make sense to mention what the allowed value ranges are for the parameters, if its not min float / max float.